src/unistd.rs

```diff
@@ -819,6 +821,431 @@ pub fn mkstemp<P: ?Sized + NixPath>(template: &P) -> Result<(RawFd, PathBuf)> {
     Ok((fd, PathBuf::from(pathname)))
 }
 
+/// Variable names for `pathconf`
+// Note: POSIX 1003.1-2008 requires all of these symbols to be
```

Comment:
Why not have this note be part of the doc comment? If you put a space between it and the first line, it might be a nice detail to have in the docs.